atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -402,6 +402,25 @@ def get_repositories(self, digests):
 
         return output_images
 
+    def get_digests_with_types(self):
+        digests_with_types = {}
+
+        if self.workflow.push_conf.docker_registries:
+            if self.workflow.tag_conf.unique_images:
+                image = self.workflow.tag_conf.unique_images[0]
+                tag = image.to_str(registry=False)
+                digests = {}
+
+                for registry in self.workflow.push_conf.docker_registries:
```

Comment:
This line (in fact to the end of this block) looks to be indented too far. Don't we want this to run even if `self.workflow.tag_conf.unique_images` is falsy?

In fact at this point I think there will always be a unique image because tag_and_push makes sure of it.

Given that, I wonder whether this should be re-written as:
```
digests_with_types = {}
digests = {}
image = self.workflow.tag_conf.unique_images[0]
tag = image.to_str(registry=false)
for registry in self.workflow.push_conf.docker_registries:
    ....
....
```

Reply:
ok will do that